rule-types/github/semgrep_secrets_scanning.yaml

```
@@ -0,0 +1,53 @@
+---
+version: v1
+release_phase: alpha
+type: rule-type
+name: semgrep_secrets_scanning
+display_name: Enable Semgrep Pre-commit hooks for detecting secrets
+short_failure_message: Semgrep Pre-commit hook is not configured for the repository
+severity:
+  value: medium
+context: {}
+description: |
+  Verifies that Semgrep Pre-commit hook is configured for the repository
+guidance: |
+  Likewise, you can configure a hook to run a semgrep scan with the semgrep hook. As an example, the following configuration would scan the files to be committed with a specified config, skipping files with unknown extensions.
+  For more information, see the [Semgrep Pre-commit](https://semgrep.dev/docs/secure-guardrails/secure-guardrails-in-semgrep) documentation.
+def:
+  in_entity: repository
+  rule_schema:
+    type: object
+    properties: {}
+  ingest:
+    type: git
+    git: {}
+  eval:
+    type: rego
+    rego:
+      type: deny-by-default
+      def: |
+        package minder
+        import future.keywords.if
+        import future.keywords.every
```

I think `import rego.v1` brings you both of these.

```suggestion
        import rego.v1
```